src/popup.html

```diff
@@ -86,8 +86,8 @@ <h4 class="i18n-content" data-i18n-message-id="popupSignUpPanelWelcome"></h4>
             <!-- CSAT Survey -->
             <div class="fx-relay-csat-survey-link-container is-hidden" role="alert" aria-labelledby="surveyLabel" aria-hidden="true">
                 <button id="survey-label" class="i18n-content fx-relay-csat-survey-link" data-i18n-message-id="popupSurveyLink" tabindex="0">
-                <button class="fx-relay-csat-survey-close-icon" aria-label="Close survey" tabindex="0">
-                    <img class="i18n-alt-tag" data-i18n-message-id="close" alt="Close icon" src="/icons/nebula-close.svg">
+                <button class="fx-relay-csat-survey-view-icon" aria-label="view survey" tabindex="0">
+                    <img class="i18n-alt-tag" data-i18n-message-id="view" alt="view icon" src="/icons/chevron-right.svg">
```

Comment:
Couple of things jump out on this `aria-label` and `alt`:

1. Shouldn't they be localised?
2. If "view icon" was read out to me, I wouldn't quite know what to do :sweat_smile: 
3. I think an `aria-label` is redundant; AFAIK the accessible name of a button is its contents, which is the image, of which the accessible name should be the `alt` text.

Reply:
It looks like other images leave `alt=""` empty. I reused the string by the text in the banner:

<img width="161" alt="image" src="https://github.com/mozilla/fx-private-relay-add-on/assets/3924990/0d87da60-ff5a-4f27-b322-7056bab1a068">

The chevron does the exact same thing as the text. It used to be a close button, but now it just drills into the survey, going into the next panel.